plugins/linux-v4l2/v4l2-output.c

```diff
@@ -230,8 +249,43 @@ static bool virtualcam_start(void *data)
 		free(list[n]);
 	free(list);
 
-	if (!success)
+	return success;
+}
+
+static bool virtualcam_start(void *data)
+{
+	struct virtualcam_data *vcam = (struct virtualcam_data *)data;
+
+	if (!loopback_module_loaded()) {
+		if (loopback_module_load() != 0) {
+			return false;
+		}
+	}
+
+	bool success = loopback_card_open(data, OBS_V4L2_CARD_LABEL);
+
+	if (!success && v4l2loopback_ctl_avaialble) {
+		// Try to dynamically add a new device using the `v4l2loopback-ctl`
+		// utility, if available and if possible. Only try this once.
+
+		// TODO: Parse the output of add command and connect directly
```

Comment:
Its a "nice to have", based on one of the previous reviews. 
From a technical point it would be nicer, but code wise its just extra LOC achieving the same functionality.

I will try to carve some time out to address the latest comments, soon.